backend/src/admin/PERMISSION_MATRIX.md

```
@@ -0,0 +1,365 @@
+  - `action`: `verify_artist`
+  - `entityType`: `artist`
+  - `previousState`: `{ verified: false }`
+  - `newState`: `{ verified: true, verifiedAt: "2024-01-15T10:30:00Z" }`
+
+**Unverify Artist**
+- **Permission**: `unverify_artists`
+- **Endpoint**: `PUT /api/admin/artists/:artistId/unverify`
+- **Audit Fields**:
+  - `action`: `unverify_artist`
+  - `entityType`: `artist`
+  - `previousState**: `{ verified: true }`
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)